### ISSUE TITLE
graphviz fails to build

### ISSUE THREAD

**Issue by muellermartin** [Homebrew/homebrew-core#10161]
graphviz fails to build from source with error "wrong number of arguments (1 for 2..3)". This happens because erroneous syntax in the Formula propably because of a wrongly applied patch in commit 06731b6 (related to issue #9086).
There the issue suggests adding following substitution in the configure script:

    s.gsub! "if test `$SWIG -php7 2>&1", "if test `$SWIG -php0 2>&1"

but the current formula only contains part of this line:

    s.gsub! "if test  -php0 2>&1"

The relevant log:
```
Error: wrong number of arguments (1 for 2..3)
Please report this bug:
  http://docs.brew.sh/Troubleshooting.html
/usr/local/Homebrew/Library/Homebrew/extend/string.rb:51:in `gsub!'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/graphviz.rb:63:in `block in install'
/usr/local/Homebrew/Library/Homebrew/utils/inreplace.rb:26:in `block in inreplace'
/usr/local/Homebrew/Library/Homebrew/utils/inreplace.rb:22:in `each'
/usr/local/Homebrew/Library/Homebrew/utils/inreplace.rb:22:in `inreplace'
/usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Formula/graphviz.rb:61:in `install'
/usr/local/Homebrew/Library/Homebrew/build.rb:137:in `block in install'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1135:in `block in brew'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1928:in `block in stage'
/usr/local/Homebrew/Library/Homebrew/resource.rb:98:in `block in unpack'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:14:in `block in mktemp'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `block in run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `chdir'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:74:in `run'
/usr/local/Homebrew/Library/Homebrew/extend/fileutils.rb:13:in `mktemp'
/usr/local/Homebrew/Library/Homebrew/resource.rb:94:in `unpack'
/usr/local/Homebrew/Library/Homebrew/resource.rb:86:in `stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1915:in `stage'
/usr/local/Homebrew/Library/Homebrew/formula.rb:1130:in `brew'
/usr/local/Homebrew/Library/Homebrew/build.rb:112:in `install'
/usr/local/Homebrew/Library/Homebrew/build.rb:189:in `<main>'
```

**Comment by ilovezfs** [comment]
oopsies

**Comment by ilovezfs** [comment]
Fixed in bdd6d0948f. Thanks for reporting this @muellermartin

**Comment by muellermartin** [comment]
@ilovezfs You're welcome and thanks for fixing it!
